tests/integration/s2n_dynamic_record_size_test.py

```diff
@@ -36,6 +36,7 @@
 test_file = './data/test_buf'
```

Comment:
I hate to be that guy..... but can you expand the comment on line 17 with a better description of what this test is attempting to do and how it does it?

Reply:
Haha no worries, I love documentation [#1e2620f](https://github.com/awslabs/s2n/pull/2292/commits/1e2620fc4a7e9085f5b52b9796bc2799ce3edc93)